driver-async/src/main/com/mongodb/async/client/MongoCollection.java

```diff
@@ -553,6 +553,9 @@
     /**
      * Executes a mix of inserts, updates, replaces, and deletes.
      *
+     * <p>Note: Supports retryable writes on MongoDB server versions 3.6 or higher. The eligibility for retryable write support for
```

Comment:
What do you think about adding

  "... or higher when the retryWrites setting is enabled."

?

Otherwise, I could see readers thinking that it will just happen automatically.  

I was also thinking about an @see link to the setting, but that's awkward now that we have multiple settings classes.

Reply:
Done